cmd/rollout-operator/main.go

```diff
@@ -24,18 +25,19 @@ import (
 func main() {
 	// CLI flags.
 	serverPort := flag.Int("server.port", 8001, "Port to use for exposing instrumentation and readiness probe endpoints.")
-	kubeAPIURL := flag.String("kubernetes.api-url", "", "The Kubernetes server API URL. If not specified, it will be auto-detected when running within a Kubernetes cluster.")
-	kubeConfigFile := flag.String("kubernetes.config-file", "", "The Kubernetes config file path. If not specified, it will be auto-detected when running within a Kubernetes cluster.")
-	kubeNamespace := flag.String("kubernetes.namespace", "", "The Kubernetes namespace for which this operator is running.")
+	kubeAPIURL := flag.String("kubernetes.api-url", "", "The API server API URL. If not specified, it will be auto-detected when running within a API cluster.")
+	kubeConfigFile := flag.String("kubernetes.config-file", "", "The API config file path. If not specified, it will be auto-detected when running within a API cluster.")
+	kubeNamespace := flag.String("kubernetes.namespace", "", "The API namespace for which this operator is running.")
+	reconcileInterval := flag.Duration("reconcile.interval", 5*time.Second, "The interval of reconciliation.")
 	logLevel := flag.String("log.level", "debug", "The log level. Supported values: debug, info, warn, error.")
 	flag.Parse()
 
 	// Validate CLI flags.
 	if *kubeNamespace == "" {
-		fatal(errors.New("The Kubernetes namespace has not been specified."))
+		fatal(errors.New("The API namespace has not been specified."))
```

Comment:
Is this change intentional? Same across the rest of the file, where doesn't look right to me.

Reply:
Definitely not intentional, reverted in https://github.com/grafana/rollout-operator/pull/30/commits/b69aaa35866a9d73aea263908fd94678d487edce.